d-hawkins-newzealand.md

```diff
@@ -0,0 +1,6 @@
+# New Zealand
+
+## Places to see in the North Island
+
+## Places to see in the South Island
```

Comment:
Yeah, everyone in New Zealand was in the film. My sister was one of the Elves, my dad was a soldier. Pretty funny really.